gradlew

```diff
@@ -161,4 +161,9 @@ function splitJvmOpts() {
 eval splitJvmOpts $DEFAULT_JVM_OPTS $JAVA_OPTS $GRADLE_OPTS
 JVM_OPTS[${#JVM_OPTS[*]}]="-Dorg.gradle.appname=$APP_BASE_NAME"
 
+# by default we should be in the correct project dir, but when run from Finder on Mac, the cwd is wrong
```

Comment:
I didn't change this, the command I ran did it: `./gradlew wrapper --gradle-distribution-url https://services.gradle.org/distributions/gradle-3.0-all.zip`